linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -78,13 +80,14 @@ object Infos {
     val isAbstract: Boolean,
     version: Version,
     byClass: Array[ReachabilityInfoInClass],
-    globalFlags: ReachabilityInfo.Flags
-  ) extends ReachabilityInfo(version, byClass, globalFlags)
+    globalFlags: ReachabilityInfo.Flags,
+    invalidLinkTimeProperties: Array[LinkTimeProperty]
```

Comment:
`Infos` must not (transitively) contain any `Position`s, in order for the algorithm to stay stable under incremental compilation. So we can't directly store the `LinkTimeProperty` node here; we must store its `(name, tpe)` pair instead.